vnext/Desktop/DevSupportManager.cpp

```diff
@@ -59,7 +59,12 @@ string DevSupportManager::GetJavaScriptFromServer(
   Url url(bundleUrl);
   auto const resolveResult = m_resolver.resolve(url.host, url.port);
   tcp::socket socket{m_context};
-  connect(socket, resolveResult);
+  boost::system::error_code ec;
+  connect(socket, resolveResult, ec);
+  if (ec) {
+    m_exceptionCaught = true;
+    return R"({"error:")"s + ec.message() + R"("})"s;
```

Comment:
>return [](start = 4, length = 6)

It's strange to return string as error. Why not just have m_errorString? Or some other error pattern?